### DISCUSSION TITLE
Temp time series review and a textbook link

### DISCUSSION THREAD

**Discussion by cmq879** [earthlab-education/Earth-Analytics-2023-01-Intro#315] [Q&A02 Open Reproducible Science]
I'm not sure where is best to post this and am combining unrelated items into a single post. 
First, for the review of other students' temperature time series assignment, are we reviewing only the code they are submitting in the readme edit, or reviewing their portfolio page with the analysis, and/or reviewing the code they committed in their own repo for the assignment? I think I'm missing where the instructions are/were. 

I think I read somewhere that we were supposed to submit any issues in the textbook that we came across. I found a dead link in the textbook at the following page: https://earthlab-education.github.io/Earth-Analytics-2023-01-Intro/intro-eds-textbook/04-python-fundamentals/01-get-started-python/python-fundamentals-01-get-started-python.html
It's under the "what you need" section, the Jupyter Notebook chapter link, this sentence: "Be sure that you have completed the chapter on [Jupyter Notebook](https://earthlab-education.github.io/Earth-Analytics-2023-01-Intro/intro-eds-textbook/04-python-fundamentals/01-get-started-python/%7B%7B%20site.url%20%7D%7D/courses/intro-to-earth-data-science/open-reproducible-science/jupyter-python/).

**Comment by eculler** [comment]
Thanks @cmq879 ! Making this an issue for myself.
